cmd/registry/cmd/compute/complexity.go

```diff
@@ -105,7 +106,8 @@ func (task *computeComplexityTask) Run(ctx context.Context) error {
 		}
 		document, err := oas3.ParseDocument(data)
 		if err != nil {
-			return fmt.Errorf("invalid OpenAPI: %s", spec.Name)
+			log.WithError(err).Warnf("Invalid OpenAPI: %s", spec.Name)
```

Comment:
This log seems to point out an error - something claims to be OpenAPI v3 but can't be parsed as OpenAPI v3. I think of warnings as "successful but worthy of attention" events. Can you elaborate on your choice?

+1 toward writing down guidelines for each log level at some point.

Reply:
Thanks, that's valid - I didn't think very hard about the "Warn" level; I mainly wanted to keep these problems from causing their parent operations to fail (so that a computation could fail for one invalid spec but continue and succeed for others). I've changed these from `Warnf` to `Errorf` and the demos still seem to run as expected, so I'll merge that and we can revisit again if needed.